onnxruntime/core/optimizer/attention_fusion_helper.h

```diff
@@ -444,6 +444,15 @@ struct AttentionMaskNodes {
   const Node* unsqueeze_1;
 };
 
+struct AttentionMaskNodesDistilBert {
+  const Node* softmax;
```

Comment:
Coding style: softmax and cast uses lower cases, which are different from other members.